types/mercadopago-sdk-js/mercadopago-sdk-js-tests.ts

```diff
@@ -32,6 +32,25 @@ brickBuilder.create('cardPayment', 'container', {
     },
 });
 
+brickBuilder.create('wallet', 'container', {
```

Comment:
Não teria que passar uma preference já que a value prop é `smart_option`?

Reply:
Not for the typing purposes, only when really implementing it.